tests/tests_pytorch/models/test_restore.py

```diff
@@ -465,6 +465,7 @@ def test_load_model_from_checkpoint(tmp_path, model_template):
 
     # Ensure that model can be correctly restored from checkpoint
     pretrained_model = model_template.load_from_checkpoint(last_checkpoint)
+    pretrained_model.to(model.device)  # Move pretrained_model to the same device as model
```

Comment:
I'm ok with this fix. A slightly better way IMO would be to set

```
"accelerator": "cpu",
```

in the `trainer_options` above. The test doesn't need to run on different accelerators.

Reply:
yeah makes sense, if the tests only run on CPU adding this option is enough here 👍